source/backend/api/Models/Concepts/DispositionFile/DispositionFileModel.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Collections.Generic;
+
+/*
+* Frontend model
+* LINK @frontend/src\models\api\DispositionFile.ts:14
```

Comment:
nit and question: are we forced to use a mix of forward slashes and backward slashes here or can we use forward slashes only?

Reply:
That would be a better question for @FuriousLlama, as currently he is the only one with this library functional. This conforms to the pattern that I've seen on other backend model files.